### DISCUSSION TITLE
I got a problem

### DISCUSSION THREAD

**Discussion by MinRabo** [LSPosed/MagiskOnWSALocal#442] [Q&A] [answered]
This is my first time see this i don't know what it mean?

/MagiskOnWSALocal$ scripts/build.sh
INFO: Architecture: x64
INFO: Release Type: retail
INFO: Magisk Version: stable
INFO: GApps Brand: MindTheGapps
INFO: GApps Variant: pico
INFO: Root Solution: magisk
Build: RELEASE_TYPE=Retail
[sudo] password for rabo:
Generate Download Links
Traceback (most recent call last):
  File "/home/rabo/MagiskOnWSALocal/scripts/generateWSALinks.py", line 31, in <module>
    from requests import Session
ModuleNotFoundError: No module named 'requests'
Build: an error has occurred, exit

Cleanup Work Directory
/MagiskOnWSALocal$

**Comment by Howard20181** [comment, marked as answer]
Run `scripts/install_deps.sh`

**Reply by MinRabo** [reply]
It's show this

 scripts/install_deps.sh
Checking and ensuring dependencies
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
Package python3-pip is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source

Package p7zip-full is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source

E: Unable to locate package patchelf
E: Package 'python3-pip' has no installation candidate
E: Unable to locate package aria2
E: Package 'p7zip-full' has no installation candidate
Dependencies: an error has occurred, exit

**Reply by Howard20181** [reply]
Install dependencies by yourself.

**Reply by MinRabo** [reply]
How to do it?

**Reply by Howard20181** [reply]
https://command-not-found.com/

**Comment by operator-name** [comment]
I've just had a similar issue. It was caused by WSL appending the windows `%PATH%` to `$PATH`, specifically I had python in my windows path. I got around this by manually setting `$PATH` to not include any windows paths.
